### PR TITLE
Add SslVerifyMode::SslVerifyFailIfNoPeerCert

### PR DESCRIPTION
**Pull request by manuels**

### CODE REVIEW

File: openssl/src/ssl/mod.rs [hunk 1]
```diff
@@ -117,7 +117,10 @@ pub enum SslVerifyMode {
     /// Verify that the server's certificate is trusted
     SslVerifyPeer = ffi::SSL_VERIFY_PEER,
     /// Do not verify the server's certificate
-    SslVerifyNone = ffi::SSL_VERIFY_NONE
+    SslVerifyNone = ffi::SSL_VERIFY_NONE,
+    /// Terminate handshake fif client did not return a certificate.
```
Comment (gkoz) [inline openssl/src/ssl/mod.rs]: `if`? :)

**Comment by manuels** [comment]
I fixed the spelling error and implemented bitwise operations on SslVerifyMode

**Comment by sfackler** [comment]
Thanks! Could you open a new version of this against the `breaks` branch? https://github.com/sfackler/rust-openssl/tree/breaks. I'd like to collect breaking changes like this there for a bit before merging them into master to avoid too many breaking releases.

**Comment by manuels** [comment]
Sure

**Pull request closed without merging**